db/schema.rb

```diff
@@ -338,7 +339,7 @@
     t.integer "age1_known"
     t.string "sex1"
     t.integer "buy1livein"
-    t.integer "buylivein"
```

Comment:
Why is this field being removed? I don't think this is correct, you might need to rebase and migrate.

Reply:
Thanks Dan, fixed now.